### ISSUE TITLE
Spectacle does not work with Office 2016 Powerpoint

### ISSUE THREAD

**Issue by svanbekhoven** [eczarny/spectacle#387]
In the Office 2016 Preview my Spectacle v0.8.10 does not work when using Powerpoint, editing slides and having mouse or keyboard focus on the slide itself or at the navigator.  It does work for Excel and Word. When I focus at the ribbon with mouse or keyboard, it does work. Would be worth investigating for when the official version of Office 2016 will be presented.

**Comment by eczarny** [comment]
Is this still an issue? It looks like others reported similar trouble in #408 but a subsequent release of Office fixed it.

**Comment by svanbekhoven** [comment]
Not an issue anymore. Solved in release 15.11.2.
